config/default_records/submission_templates/014_lcm_triomics_submission_templates.wip.yml

```diff
@@ -0,0 +1,9 @@
+# Submission template for LCM Triomics DNA WGS and EMSeq
+---
+# LCM Triomics DNA Prep submission template
+Limber-Htp - LCM Triomics:
```

Comment:
Again, I think the submission template name should probably reference 'EM-Seq'

Reply:
Change to `Limber-Htp - LCM Triomics EMSeq:`